shared/js/utilities.js

```diff
@@ -159,6 +159,14 @@ var HtmlHelper = {
     ids.forEach(function(id) {
       context[StringHelper.camelCase(id)] = document.getElementById(id);
     });
+  },
+
+  preventPointerEvents: function(element) {
+    element.classList.add('disable-when-dialog-animates');
+    window.addEventListener('transitionend', function te_prevent() {
+      element.classList.remove('disable-when-dialog-animates');
+      window.removeEventListener('transitionend', te_prevent);
+    });
```

Comment:
I don't understand this :) we can directly add the class to the button in the markup? We don't need to add/remove it, it can stay :)